### PR TITLE
CSM 9/10 dental -> direct

### PR DESCRIPTION
**Pull request by woodbe**
CSM 9: Curved PAI mold preparation

Proposal: As there can be more materials for mold in the future, suggest to put it as general as the preparation of cast.

CSM 10: Dental Impression Material

Proposal: As there can be more materials for mold in the future, suggest to put Dental Impression in the Fingerprint Toolbox Inventory instead in the Fingerprint Toolbox Overview

From spreadsheet

### CODE REVIEW

**Comment by woodbe** [comment]
I have combined these by making the dental impression an example in the overview and using "direct" in the example itself.

**Review by n-kai — CHANGES_REQUESTED** [review]
The term "Dental impression materials" is also used in other sections like 2.2.1.1.1 and 2.2.2.1.1. They should also be changed.

File: FINGERPRINT_Toolbox_Overview.adoc [hunk 1]
```diff
@@ -144,19 +144,19 @@ The binarized images are printed onto either a sheet of laser printer transparen
 
 ===== Overview
 
-A mold is made from the target finger friction ridge pattern using dental impression material. The friction ridges of the enrolled finger are pressed into impression material to create a curved mold of the fingerprint.
+A mold is made from the target finger friction ridge pattern using a direct impression material (such as a direct impression material). The friction ridges of the enrolled finger are pressed into impression material to create a curved mold of the fingerprint.
```
Comment (gfiumara) [inline FINGERPRINT_Toolbox_Overview.adoc]:
```suggestion
A mold is made from the target finger friction ridge pattern using a direct impression material (such as a dental impression material). The friction ridges of the enrolled finger are pressed into impression material to create a curved mold of the fingerprint.
```

**Review by gfiumara — APPROVED** [review]

**Review by n-kai — CHANGES_REQUESTED** [review]
It's better to change L147

a direct impression material (such as a direct impression material)
->
a direct impression material (such as a **dental** impression material)

**Pull request merged**